src/index.tsx

```diff
@@ -60,24 +60,24 @@ function Updaters() {
 
 ReactDOM.render(
   <StrictMode>
-    <LanguageProvider>
-      <FixedGlobalStyle />
-      <Web3ReactProvider getLibrary={getLibrary}>
-        <Web3ProviderNetwork getLibrary={getLibrary}>
-          <Blocklist>
-            <Provider store={store}>
-              <Updaters />
-              <ThemeProvider>
-                <ThemedGlobalStyle />
-                <HashRouter>
+    <FixedGlobalStyle />
+    <Web3ReactProvider getLibrary={getLibrary}>
+      <Web3ProviderNetwork getLibrary={getLibrary}>
+        <Blocklist>
```

Comment:
maybe we just need to move the router/provider up instead of the language provider down? neither of those rely on the web3 react provider and the blocklist is also a candidate for translation